src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -83,23 +83,26 @@
  */
 public class EntityCitizen extends EntityAgeable implements INpc
 {
-    private static final float CONST_BED_HEIGHT = 0.6875f;
-    private static final float CONST_HALF_BLOCK = 0.5f;
+    private static final float CONST_BED_HEIGHT             = 0.6875f;
+    private static final float CONST_HALF_BLOCK             = 0.5f;
     private static final float CONST_SLEEPING_RENDER_OFFSET = -1.5f;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "f" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS818)